Source/PyFR/ColorTable.cu

```diff
@@ -1,17 +1,17 @@
 
 #include "ColorTable.h"
 
-static ::thrust::system::cuda::vector< Color> PaletteVec;
-static ::thrust::system::cuda::vector< float> PivotsVec;
-static std::vector<Color> RuntimePalette;
-static std::vector<float> RuntimePivots;
-
-RuntimeColorTable::RuntimeColorTable(FPType cmin, FPType cmax,
-  const std::vector<Color>& palette,
-  const std::vector<float>& pivots)
+static ::thrust::system::cuda::vector< Color> PaletteVec[2];
+static ::thrust::system::cuda::vector< float> PivotsVec[2];
+static std::vector<Color> RuntimePalette[2];
+static std::vector<float> RuntimePivots[2];
+
+RuntimeColorTable::RuntimeColorTable(int pipeline, FPType cmin, FPType cmax,
+  const std::vector<Color>& palette, const std::vector<float>& pivots)
```

Comment:
Instead of global static arrays that have two elements, would it make sense for `RuntimeColorTable` to own these?  Was that what was causing the issue before?  Perhaps `RuntimeColorTable` could own both the `thrust::system::cuda::vector` *and* the `std::vector` (I am assuming it only owned the thrust one before, when you were hitting issues)?  Forgive me if I'm misunderstanding what the true issue was...